lib/config.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+
+var config = require('bedrock').config;
+var path = require('path');
+
+config['bedrock-agreement-http'] = {};
+config['bedrock-agreement-http'].basePath = '/agreement';
```

Comment:
I recommend we change to `.routes.basePath` to keep that stuff better namespaced.